tests/integration/courses.test.js

```diff
@@ -3335,6 +3345,41 @@ describe('COURSES ROUTES - DELETE /courses/{_id}/trainee/{traineeId}', () => {
     });
   });
 
+  describe('HOLDING_ADMIN', () => {
+    it('should return 200 if holding admin delete learner (intra)', async () => {
```

Comment:
Que penses-tu de juste mettre "should delete course trainee" comme c'est ecrit pour les tests ROF ?

Reply:
done